packages/server-core/src/social/invite/invite.resolvers.ts

```diff
@@ -74,6 +80,16 @@ export const inviteExternalResolver = resolve<InviteType, HookContext>({
   }),
   invitee: virtual(async (invite, context) => {
     if (invite.inviteeId) return await context.app.service(userPath).get(invite.inviteeId)
+    else if (invite.token) {
+      const identityProvider = (await context.app.service(identityProviderPath).find({
+        query: {
+          token: invite.token
+        }
+      })) as Paginated<IdentityProviderType>
+      if (identityProvider.data.length > 0) {
+        invite.invitee = await context.app.service(userPath).get(identityProvider.data[0].userId)
```

Comment:
You dont have to set `invite.invitee` rather you have to return from resolver

Reply:
implemented